knowledge-base/common-null-value-parameter-format.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Value cannot be null. (Parameter 'format')
+description: Value cannot be null. (Parameter 'format')
+type: troubleshooting
+page_title: Value cannot be null. (Parameter 'format')
+slug: common-kb-null-value-parameter-format
+position: 
+tags: telerik, blazor, null, value, parameter, format, localization
+ticketid: 1528567
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>UI for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am trying to use a Wizard component in an application that has localization enabled but I am getting an error.
+
+## Error Message
+
+>warning Unhandled exception rendering component: Value cannot be null. (Parameter 'format')
+System.ArgumentNullException: Value cannot be null. (Parameter 'format')
+at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
+at System.String.Format(String format, Object arg0, Object arg1)
+at Telerik.Blazor.Components.TelerikWizard.get_PagerMessage()
+at Telerik.Blazor.Components.TelerikWizard.BuildRenderTree(RenderTreeBuilder __builder)
+at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder batchBuilder, RenderFragment renderFragment)
+at Microsoft.AspNetCore.Components.RenderTree.Renderer.RenderInExistingBatch(RenderQueueEntry renderQueueEntry)
+at Microsoft.AspNetCore.Components.RenderTree.Renderer.ProcessRenderQueue()
+
+
+## Cause\Possible Cause(s)
+
+You may run across an error similar to the above listed one for other components as well. It is associated with incomplete localization implementation and occurs if a component is used in a localized application but the resex files are missing the needed keys for the corresponding component.
+
+The app needs to either implement localization for the Telerik components correctly or not implement a service for that at all. So, it is essential that the resx files used for the Telerik components in the localized application are always up-to-date containing all the necessary keys.
+
+## Solution
+
+To solve the issue, make sure that:
+
+* The application is correctly configured to use localization - read more [here]({%slug globalization-localization%})
```

Comment:
Don't use "here" and similar meaningless keywords for links. This is bad for SEO and bad for accessibility.

```suggestion 
* The [application is correctly configured to use localization]({%slug globalization-localization%})
```